json_home_client/client.py

```diff
@@ -127,7 +127,7 @@ def content_type(self) -> MimeType:
     @property
     def encoding(self) -> str:
         """Get encoding of response."""
-        content_type = self.headers.get('content-type') if (self.headers) else None
+        content_type = self.headers['content-type'] if (self.headers) else None

```

Comment:
This should likely stay as it is; the old code gracefully returned None if 'content-type' wasn't in the headers, but the new code will panic.

Reply:
OK, thanks — reverted that change